doc/javacomplete.txt

```diff
@@ -315,6 +315,11 @@ A single letter indicates the kind of compeltion item. These kinds are:
         let g:JavaComplete_UsePython3 = 1
         an option to make javacomplete2 use python3 interpreter instead of python2, default value is 0.
 
+	Disable javacomplete2 default mappings:
```

Comment:
Seems wrong indentation.
Can you, please, ident lines the same as previous options in doc:

1. let NameOfParameter
2. descsription text.